src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -236,6 +249,7 @@ class SidebarPanels extends React.Component<Props, State> {
                         exact
                         path={`/${SIDEBAR_VIEW_BOXAI}`}
                         render={() => {
+                            this.handlePanelRender(SIDEBAR_VIEW_BOXAI);
```

Comment:
What happens if the LoadableBoxAISidebar does not load for some reason? Do we still want to call the call back before the panel loads like this?

Reply:
Actually we do as we still want to track which of the panels would be open if it loaded successfully. At this point we know which one would the active one.